src/main/proto/netflix/titus/titus_vpc_api.proto

```diff
@@ -32,6 +32,7 @@ message GetAllocationRequest {
 
 message GetAllocationResponse {
     AddressAllocation addressAllocation = 1;
+    SignedAddressAllocation signedAddressAllocation = 2;
```

Comment:
Why not replace `AddressAllocation` with `SignedAddressAllocation` since that's already a subfield?

Reply:
No reason to deprecate something early?

Reply:
Yeah, just seems potentially redundant/confusing. If we plan of deprecating or removing, seems like earlier is better but I'll defer to whichever approach you prefer.